arch/xtensa/src/esp32/Kconfig

```diff
@@ -2194,6 +2194,10 @@ config ESP32_BLE_MAX_CONN
 	range 1 3
 	default 1
 
+config BT_CTRL_HOST_FLOW_CONTROL_SUPPORT_DISABLED
+    bool
+    default y
+
```

Comment:
Please include ---help--- message. Explain clearly what is this feature and why is this needed?

Reply:
Oops, sorry, it is an intermediate flag that is not visible to the user, please ignore my previous comment